lib/ansible/plugins/action/net_template.py

```diff
@@ -93,6 +93,16 @@ def _handle_template(self):
         except IOError:
             return dict(failed=True, msg='unable to load src file')
 
+        # Create a template search path in the following order:
+        # [working_path, _role_path, *_block._role_paths, dirname(source)]
+        searchpath = [working_path]
+        if self._task._role is not None:
+            searchpath.append(self._task._role._role_path)
+            if self._task._block._dep_chain:
+                for role in sorted(self._task._block.get_dep_chain()):
```

Comment:
Thank you @bcoca and @Qalthos for the immediate reply!  And yes, @bcoca is right that I shouldn't say that was a "suggestion"... Sorry about the confusion.

By the way, what is the simplest playbook/roles to execute the following two lines, as my use case doesn't execute those two lines.  Thank you for your patience, as I'm still learning this ansible internal architecture:

+            if self._task._block._dep_chain:
+                for role in sorted(self._task._block.get_dep_chain()):

Thank you for your support!